# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/zig/hotfix-cnonshared.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

https://github.com/ziglang/zig/pull/10925#issuecomment-1046123726

--- zig-0.13.0/build.zig.vanilla	2024-06-11 10:11:46.333160416 +0200
+++ zig-0.13.0/build.zig	2024-06-11 10:12:11.433330173 +0200
@@ -707,6 +707,7 @@
                     else => |e| return e,
                 };
                 exe.linkSystemLibrary("unwind");
+                exe.linkSystemLibrary("c_nonshared");
             },
             .ios, .macos, .watchos, .tvos, .visionos => {
                 exe.linkLibCpp();
